### DISCUSSION TITLE
Video 360 from GoPro

### DISCUSSION THREAD

**Discussion by tenistik** [mediacms-io/mediacms#576] [Q&A]
Hi !
Does MediaCMS support video 360 from GoPro ?
Instead of 360 I get this:
![mediacms](https://user-images.githubusercontent.com/68004865/203062407-ef6d76c5-4af5-46f4-8576-ec5bf0eff203.png)
Thanks in advance for your reply !

**Comment by mgogoulos** [comment]
Hi, MediaCMS is using video.js for a video player, so this is the way the video player shows this type of videos. 
So it seems that video.js requires a separate plugin to handle 360 videos - https://github.com/videojs/videojs-vr . It should not be that hard to integrate!

**Reply by tenistik** [reply]
Thanks for the help !
Can you tell me more how to integrate this plugin ? Maybe a link to the manual ?
